### ISSUE TITLE
Legend become inactive with Annotations and overflow: visible

### ISSUE THREAD

**Issue by kutensky** [highcharts/highcharts#9217]
I configured chart with annotations and legend. We use "overflow: visible" for chart. And in that case when you hide some series by clicking on legend item, legend items become inactive, so you can't show series again.
It happens because when you hide some series, annotation hides by moving to y = -9000. And because "overflow" is "visible", chart container blocks chart content.

#### Expected behaviour
https://jsfiddle.net/LLExL/13258/

#### Actual behaviour
https://jsfiddle.net/LLExL/13257/

**Comment by pawelfus** [comment]
Thanks fore reporting!

**Comment by kutensky** [comment]
As a workaround, I updated annotations.js.
I've added next code in redrawItem inside "if (item-itemType === 'label')" block:
```
var arePointsHidden = points.every(function(p) {
	return p.options.visible === false || p.series.options.visible === false;
});

arePointsHidden ? item.hide() : item.show();
```
and in "alignItem" i commented next code:
```
// item.attr({
//     x: 0,
//     y: -9e9
// });
```

**Comment by TorsteinHonsi** [comment]
Thanks @kutensky !

@pawelfus Feel free to assign further.

**Comment by ppotaczek** [comment]
After debugging, I noticed that this problem occurs only in Chrome browser. If we remove CSS styles and click on the legend item, the entire chart will disappear: https://jsfiddle.net/BlackLabel/cnaLjzge/

This bug is related with large translate value: https://jsfiddle.net/BlackLabel/2cqg7k4u/

Reported issue: https://bugs.chromium.org/p/chromium/issues/detail?id=911186

**Comment by pawelfus** [comment]
@ppotaczek issue on github is closed and fixed. But we still can observe the same behaviour in Chrome v72 - could you confirm?

**Comment by ppotaczek** [comment]
@pawelfus, 

Yes I can confirm. Comments on the chromium issue are a bit unclear, but it seems that the fix will be applied in Chrome version 73.0.3628.0

**Comment by pawelfus** [comment]
Just tested with Chrome: Version 73.0.3683.75 (Official Build) (64-bit) - works fine. Closing the issue, if this reappears, let us know.
